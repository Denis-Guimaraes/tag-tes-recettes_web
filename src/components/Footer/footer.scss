@import 'src/styles/vars';

.footer {
  padding: $base-padding;
  background-color: $color-dark;
  color: $color-light;
  display: flex;
  justify-content: space-between;
  align-items: center;

  &__link {
    padding: $base-padding/2;
    font-size: $base-font-size*0.9;
    text-decoration: underline; 
  }
}
