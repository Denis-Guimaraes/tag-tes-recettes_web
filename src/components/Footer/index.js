// NPM import
import React from 'react';

// Local import

// Styles
import './footer.scss';

// Code
const getYear = () => {
  const date = new Date();
  return date.getFullYear();
};

const Footer = () => (
  <footer className="footer">
    <p className="footer__text">&copy; {getYear()} Denis Guimaraes.</p>
    <ul>
      <li className="footer__link">Mentions l√©gales</li>
      <li className="footer__link">Contact</li>
    </ul>
  </footer>
);

// Export
export default Footer;
