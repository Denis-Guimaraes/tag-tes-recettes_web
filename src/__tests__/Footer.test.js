/* eslint-disable no-undef */
// NPM import
import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
// Local import
import Footer from '../components/Footer';


describe('Footer', () => {
  it('match snapshot', () => {
    const wrapper = shallow(<Footer />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
