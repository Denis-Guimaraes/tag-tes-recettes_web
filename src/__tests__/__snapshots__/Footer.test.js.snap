// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer match snapshot 1`] = `
<footer
  className="footer"
>
  <p
    className="footer__text"
  >
    © 
    2019
     Denis Guimaraes.
  </p>
  <ul>
    <li
      className="footer__link"
    >
      Mentions légales
    </li>
    <li
      className="footer__link"
    >
      Contact
    </li>
  </ul>
</footer>
`;
